src/components/VirtualizedDataTable/VirtualizedDataTable.tsx

```diff
@@ -0,0 +1,185 @@
+import { useRef } from 'react'
+import classNames from 'classnames'
+import { useVirtualizer, VirtualItem, Virtualizer } from '@tanstack/react-virtual'
+import { Loader } from '../Loader/Loader'
+import type { ColumnConfig, Column } from '../DataTable/DataTable'
+
+const DEFAULT_ROW_HEIGHT = 52
+const DEFAULT_OVERSCAN = 5
+const DEFAULT_TABLE_HEIGHT = 500 // Shows ~9-10 rows at default row height
```

Comment:
For `DEFAULT_TABLE_HEIGHT`, to get exactly 10 rows, you'll want:
```suggestion
const DEFAULT_NUM_ROWS = 10
const DEFAULT_TABLE_HEIGHT = (DEFAULT_ROW_HEIGHT * DEFAULT_NUM_ROWS) + HEADER_HEIGHT - 1
```